<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="pdfPathInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>84</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter PDF File path of result</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 3px;
border-color: royalblue;
border-width: 2px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="obtNumberInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>167</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter Your Obtained Number </string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 3px;
border-color: royalblue;
border-width: 2px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pdfPathLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter PDF Path:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>84</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Browse PDF File of Result</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 3px;
border-color: royalblue;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="obtnumberlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>144</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Your Obtained Number:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>237</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>16</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 3px;
border-color: royalblue;
border-width: 2px;</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="myResultCheck">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>209</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>This will display only your result</string>
   </property>
   <property name="text">
    <string>Display My Result Only</string>
   </property>
  </widget>
  <widget class="QListWidget" name="allResultListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="allResultsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>275</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All Results:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yourResultLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>512</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your Result:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="yourResultListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>533</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 3px;
border-color: royalblue;
border-width: 2px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>207</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Select Your Class/Grade</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>XI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XII</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="selectClassLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>209</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Your Class:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Note">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>123</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Condensed</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>NOTE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="noteLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Wrong time, right place</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="BlueFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: royalblue;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="OnTopWaitLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Condensed</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Ready...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
